CONTRIBUTING.md

```diff
@@ -60,25 +62,34 @@ This section describes how to build and run the Botkube from the source code.
      docker tag ghcr.io/kubeshop/botkube:v9.99.9-dev-amd64 <your_account>/botkube:v9.99.9-dev
      docker push <your_account>/botkube:v9.99.9-dev
      ```
-     Where `<your_account>` is Docker hub account to which you can push the image.
+     Where `<your_account>` is Docker hub or any other registry provider account to which you can push the image.
+
+2. Ensure you have a Slack installation already setup:
 
-2. Deploy the newly created image in your cluster:
+   Please refer to the [Slack installation instructions](https://botkube.io/docs/installation/socketslack/) to ensure you have a valid integration.
 
+3. Deploy the newly created image in your cluster:
+   
    ```sh
    helm install botkube --namespace botkube --create-namespace \
-   --set communications.slack.enabled=true \
-   --set communications.slack.channel=<SLACK_CHANNEL_NAME> \
-   --set communications.slack.token=<SLACK_API_TOKEN_FOR_THE_BOT> \
-   --set settings.clusterName=<CLUSTER_NAME> \
-   --set settings.kubectl.enabled=<ALLOW_KUBECTL> \
-   --set image.registry=<image_registry e.g. docker.io> \
+   --set communications.default-group.socketSlack.enabled=true \
+   --set communications.default-group.socketSlack.channels.default.name=<SLACK_CHANNEL_NAME> \
+   --set communications.default-group.socketSlack.appToken=<SLACK_API_APP_TOKEN> \
+   --set communications.default-group.socketSlack.botToken=<SLACK_API_BOT_TOKEN> \
+   --set settings.clusterName=${CLUSTER_NAME} \
+   --set executors.kubectl-read-only.kubectl.enabled=true \
+   --set image.registry=<IMAGE_REGISTRY e.g. docker.io> \
    --set image.repository=<your_account>/botkube \
    --set image.tag=v9.99.9-dev \
+   --set image.pullPolicy=<IMAGE_PULL_POLICY e.g. Always>
```

Comment:
What do you think to get rid of the full command (which would be hard to maintain + it works with Slack only), and instead say "hey, follow the Slack/Discord etc. installation docs as usual, but during Helm installation provide additional flags to override the image".

And there would be just the flags listed:
```
   --set image.registry=<IMAGE_REGISTRY e.g. docker.io> \
   --set image.repository=<your_account>/botkube \
   --set image.tag=v9.99.9-dev \
   --set image.pullPolicy=<IMAGE_PULL_POLICY e.g. Always>
```

Reply:
That was my initial thinking too. But, the duplication here makes it a bit easier for contributors to just get up and running.

Reply:
...As long as the commands are up to date, and as long as the contributor wants to develop Botkube against Slack 🙂 All the options are well explained under installation docs, and the description is missing from here. So everybody still will end up navigating to the installation docs. I don't think passing additional parameters during the `helm install` is any harder than this 🤔 
While I'd still vote for the simpler option, if you don't agree it's totally alright - let's create a quick poll on Slack and select the approach the whole team likes the most 👍